.github/workflows/stale.yml

```diff
@@ -0,0 +1,31 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 90
```

Comment:
So this deletes any issues that haven't had any activity for the last 90 days.  During one of our meetings, I though we'd decided to make Jan 1, 2022 the cutoff date.  Can this work with dates or only days?

Reply:
Good point! We changed it for the `shopify_app` repo to 90 days but I can change it to the start of the year for the rest of the repos. I couldn't find any obvious way to do it by date instead of number of days, so I think the best thing to do would change the number of days to like 264 which would be the start of the year.

Reply:
Yeah, was trying to figure out if we could set it with `days-before-issue-stale: ${{ some expression code here }}` but it's probably more hassle than the effort required!

What I was going to do was set it to `-1` and then have a script that sets the `Stale` label on issues that haven't had activity since date X